rep-2000.rst

```diff
@@ -709,6 +709,121 @@ Package manager use for dependencies:
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS   | Tier 2        | All Platforms               | All Architectures              |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++17
+- Python 3.6
+
+
+Dependency Requirements:
+
++-------------+---------------------------------+------------------------------------------------------+
+|             |        Required Support         |                 Recommended Support                  |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Package     |  Ubuntu Focal  |   Windows 10** |     MacOS**   | Debian Buster  | OpenEmbedded**      |
++=============+================+================+===============+================+=====================+
+| CMake       |     3.16.3     |     3.19.1     |     3.14.4    |      3.13.4    | 3.16.1 / 3.12.2**** |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| EmPY        |                                      3.3.2                                             |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Gazebo      |     11.0.0*    |      N/A       |     11.0.0    |      11.0.0*   |        N/A          |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Ignition    |    Edifice*    |      N/A       |            Edifice*            |        N/A          |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Ogre        |                                      1.10*                       |        N/A          |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| OpenCV      |     4.2.0      |     3.4.6*     |     4.2.0     |      3.2.0     |  4.1.0 / 3.2.0****  |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| OpenSSL     |     1.1.1d     |     1.1.1i     |     1.1.1f    |      1.1.1d    | 1.1.1d / 1.1.1b**** |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Poco        |     1.9.2      |     1.8.0*     |     1.9.0     |      1.9.0     |        1.9.4        |
++-------------+----------------+----------------+---------------+----------------+---------------------+
```

Comment:
From what I can tell, we've dropped the poco dependency in ROS 2. It's no longer part of the `ros2.repos` file at all. Can this be dropped?

Related to ros2/ros2#867

```suggestion
```

Reply:
Yes, I think that's correct.

Reply:
Good catch, yes, we can drop that.